### ISSUE TITLE
Update Example Documentation

### ISSUE THREAD

**Issue by alexjc** [cjw296/sybil#14]
The first example from the documentation does not include the `CodeBlockParser()`, so doesn't pick up `code-block` directives.
https://sybil.readthedocs.io/en/latest/use.html

Here's what I'm using:

```
from doctest import ELLIPSIS
from sybil import Sybil
from sybil.parsers.codeblock import CodeBlockParser
from sybil.parsers.doctest import DocTestParser
from sybil.parsers.skip import skip
from sybil.parsers.capture import parse_captures

pytest_collect_file = Sybil(
    parsers=[
        CodeBlockParser(),
        DocTestParser(optionflags=ELLIPSIS),
        parse_captures,
        skip,
    ],
    pattern="test_*.rst",
).pytest()
```

P.S. Nice library!

**Comment by cjw296** [comment]
Sure, would you be able to work up a pull request to fix this?

**Comment by cjw296** [comment]
Closed by #15

**Comment by alexjc** [comment]
That was fast. Thank you!
